sql/src/main/java/io/crate/protocols/postgres/types/VarCharType.java

```diff
@@ -60,8 +59,7 @@ public int writeAsText(ByteBuf buffer, @Nonnull Object value) {
 
     @Override
     protected byte[] encodeAsUTF8Text(@Nonnull Object value) {
-        assert value instanceof String : "value must be a string";
```

Comment:
Why can we have different types here now?

Reply:
Because the UndefinedType is mapped to a PG VarCharType and thus not only strings will end up here.

Reply:
Is `toString` safe for all types ? E.g. what if it is an array or an object itself?

Reply:
Yes right, I was also thinking that this should be fixed properly maybe in a dedicated commit/PR as it is not really related to this one.

Reply:
Calling `toString` is safe on any instance but it won't result in useful content. As this is more a fallback than the usual case, I'm not sure how far we wanna go here with arrays/list/maps/objects. What do you think?

Reply:
I've removed this change as it is not related to this PR.